src/harrypotterplugin/utilities/PlayerStats.java

```diff
@@ -1,16 +1,21 @@
 package harrypotterplugin.utilities;
 
-import java.io.File;
-import java.io.IOException;
-import java.util.HashMap;
+import lombok.Getter;
+import lombok.Setter;
 import org.bukkit.Bukkit;
 import org.bukkit.configuration.file.FileConfiguration;
 import org.bukkit.configuration.file.YamlConfiguration;
 import org.bukkit.entity.Player;
 
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+
 public class PlayerStats {
 	
 	private final Player player;
+	@Getter
```

Comment:
There is no need to use this library you can auto generate getter and setter in eclipse, this is unneeded overhead